package.json

```diff
@@ -16,7 +16,9 @@
     "safari"
   ],
   "author": "Vojta Jina <vojta.jina@gmail.com>",
-  "dependencies": {},
+  "dependencies": {
+    "wd": "https://github.com/admc/wd.git#master"
```

Comment:
It should be okay to use the latest npm release of `wd` (v1.12.1) now, correct?

Reply:
At work we have it locked against [this commit: c7d022f10a3585512f7d0892c8e33925df68d4c6 in wd](https://github.com/admc/wd/commit/c7d022f10a3585512f7d0892c8e33925df68d4c6).

That commit is contained within v1.11.2, v1.11.3, v1.11.4, v1.12.0 and v1.12.1. I've added a new commit that puts the dependency at `>=1.11.2`.